docs/usage/Transaction.md

```diff
@@ -0,0 +1,480 @@
+  
+**Parameters**:
+
+| parameter                                | type            | required           | Description                                                                                                                                                                    |  
+|------------------------------------------|-----------------|--------------------| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **unsafe**                               | Boolean/Object  | yes                | unsafe if true, skip all tests. if it's an object, it's expected to contain a set of flags to skip certain tests.                                                                                                           |
+
+**Returns**: {string}
+
+## .checkSerialize(opts)
+**Description**: Retrieve a hex string that can be used with dashd's CLI interface (decoderawtransaction, sendrawtransaction)
+
+**Parameters**:
+
+| parameter                                | type            | required           | Description                                                                                                                                                                    |  
+|------------------------------------------|-----------------|--------------------| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **opts**                                 | Boolean/Object  | yes                | opts allows to skip certain tests. {see Transaction#serialize}.                                                                                                           |
+
+**Returns**: {string}
+
+## .getSerializationError(opts)
+**Description**: Retrieve a possible error that could appear when trying to serialize and broadcast this transaction.
+
+**Parameters**:
+
+| parameter                                 | type            | required           | Description                                                                                                                                                                    |  
+|-------------------------------------------|-----------------|--------------------| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **opts**                                  | Boolean/Object  | yes                | opts allows to skip certain tests. {see Transaction#serialize}.                                                                                                           |
+
+## .lockUntilDate(time)
+**Description**: Sets nLockTime so that transaction is not valid until the desired date(a timestamp in seconds since UNIX epoch is also accepted)
```

Comment:
```suggestion
**Description**: Sets nLockTime so that transaction is not valid until the desired date (a timestamp in seconds since UNIX epoch is also accepted)
```